src/components/shared/HamburgerMenu.tsx

```diff
@@ -26,7 +26,11 @@ const HamburgerMenu: React.FC = () => {
   };
 
   useEffect(() => {
-    setCurrentLocation(location.pathname);
+    if (location.pathname == '/bookings/B1') {
```

Comment:
Why so?

Reply:
To keep the user's navigation active on booking page. although I forget to make it dynamic. 
updated one :
useEffect(() => {
    if (/^\/bookings\/[^/]+$/.test(location.pathname)) {
      setCurrentLocation('/bookings');
    } else {
      setCurrentLocation(location.pathname);
    }
  }, [location]);